.circleci/config.yml

```
@@ -78,41 +78,84 @@ jobs:
           name: Run jasmine tests (part C)
           command: ./.circleci/test.sh no-gl-flaky-jasmine
 
-  stable-image:
-    docker:
-      - image: plotly/testbed:latest
+  make-baselines:
     parallelism: 4
-    working_directory: /var/www/streambed/image_server/plotly.js/
+    docker:
+      - image: circleci/python:3.8.9
+    working_directory: ~/plotly.js
     steps:
       - attach_workspace:
-          at: /var/www/streambed/image_server/
+          at: ~/
+      - run:
+          name: which pip3 version
+          command: which pip3 && pip3 --version
```

```suggestion
          name: get latest pip
          command: pip install --upgrade pip
```
You're on the `circleci/python:3.8.9` image so the only Python/pip you have should be from Py3.8 - but there have been big changes in pip recently that it's prudent to keep up with.

---

Discussing the "test environment" with @jonmmease and @nicolaskruchten last week, we thought it would be better to pin down all the dependency versions on this container to avoid potential surprises (if any).
But if we want to test "latest" `kaleido` and `plotly.io` with this image test, then yes we could use latest versions.

---

Agreed that we should pin `kaleido` and `plotly`, I'm just talking saying we should be using the latest version of `pip` itself.

---

If/when the `pip` version used by container `circleci/python:3.8.9` become too old, we can simply bump the container not the `pip` version. Can't we?

---

ah ok, it's 21.0.1 in this container, that should be fine. The key changes happened in late v20.